src/Kusto/Kusto/Commands/UpdateAzureRmKustoCluster.cs

```diff
@@ -55,12 +55,17 @@ public class UpdateAzureRmKustoCluster : KustoCmdletBase
         [PSArgumentCompleter("KC8", "KC16", "KS8", "KS16", "L8", "L16", "D14_v2", "D13_v2")]
         public string SkuName { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The instance number of the VM.")]
+        public int? Capacity { get; set; }
+
         [Parameter(
             Mandatory = false,
             HelpMessage = "Name of the Tier used to create the cluster")]
         [PSArgumentCompleter("Standard")]
         public string Tier { get; set; }
```

Comment:
You can make any breaking change you want as long as you are still in preview.  Feel free to remove.

Reply:
Thanks